crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/builder/CrownCourtDTOBuilder.java

```diff
@@ -18,7 +18,16 @@
 public class CrownCourtDTOBuilder {
 
     public static CrownCourtDTO build(final ApiProcessRepOrderRequest request) {
-        CrownCourtDTO.CrownCourtDTOBuilder builder = CrownCourtDTO.builder()
+        CrownCourtDTO.CrownCourtDTOBuilder builder = mapToCrownCourtDTO(request);
+
+        if (request instanceof ApiUpdateApplicationRequest updateRequest) {
+            mapUpdateApplicationRequestToCrownCourtDTO(updateRequest, builder);
```

Comment:
Can't we just add another condition here to build the `UpdateCrownCourt` specific fields?